### PR TITLE
fix(tauri): http jellyfin server address

### PR DESCRIPTION
**Pull request by endrl**
Disable https context.

Closes: #2009

### CODE REVIEW

**Review by prayag17 — COMMENTED** [review]
You don't need to use setup to create a new window, you can still use the older window created in tauri.conf.json, also you should disable the localhost server when in dev mode since vite already does that

**Comment by endrl** [comment]
Updated, also added save/restore of window state which is probably more convenient

**Comment by endrl** [comment]
Tauri merged a new option to enable http for windows. This supersedes the localhost plugin. https://github.com/tauri-apps/tauri/pull/7645

**Comment by ferferga** [comment]
@endrl That's great news! As soon as I see a Tauri update from renovate I'll merge so you can update this PR accordingly :)

**Comment by ferferga** [comment]
@endrl Can you try the artifacts of #2096 and let me know if merging that PR fixes this?

**Comment by prayag17** [comment]
I will have to check if their fix works on Linux before merging it.

**Comment by endrl** [comment]
Oh, already released. That's fast.

I installed the deb and i noticed:
- App Icon is not available
- Loading screen loops endless. It might be the CSP (console is not enabled!)

So we are back at: https://github.com/jellyfin/jellyfin-vue/pull/2049

**Comment by ferferga** [comment]
@endrl But it's the original issue that ti's PR was addressing solved?

**Comment by endrl** [comment]
Oh, there is missing flag. Now it's working. You can merge this and update to tauri 1.5

**Review by ferferga — APPROVED** [review]

File: tauri/tauri.conf.json [hunk 1]
```diff
@@ -52,7 +52,8 @@
       }
     ],
     "security": {
-      "csp": "default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'"
+      "csp": "default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'",
+      "dangerousUseHttpScheme": true
     }
   }
-}
+}
```
Comment (ferferga) [inline tauri/tauri.conf.json:59]:
```suggestion
}

```

**Comment by ferferga** [comment]
@endrl The checos are still failing for some reason

**Comment by ferferga** [comment]
@endrl Oops, good catch! Let's rerun checks after the renovate PR.

**Pull request merged**